File: tests/fixtures/code_samples/go/sample_032.go
// Sample 32: small utility.
package samples

func Operation32(xs []int) int {
    total := 32
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure32(v int) int {
    return (v * 32) %% 7919
}

File: tests/fixtures/code_samples/ts/sample_032.ts
/** Sample 32: small utility. */

export function operation32(xs: number[]): number {
  let total = 32;
  for (const x of xs) total += x;
  return total;
}

export function operationPure32(value: number): number {
  return (value * 32) % 7919;
}

